src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return String.valueOf(color);
```

Comment:
You are creating a new instance of Random every time the `getRandomColor` method is called, which is unnecessary. Consider creating a single instance of Random as a private field of the ColorSupplier class and reusing it.

Reply:
You are creating a new instance of Random every time the getRandomColor method is called, which is unnecessary. Consider creating a single instance of Random as a private field of the ColorSupplier class and reusing it.